docker-compose.override.yml

```diff
@@ -7,11 +7,9 @@ services:
       - ACCEPT_EULA=Y
     ports:
       - "5433:1433"   
-
-  cache:
+  redis:
```

Comment:
We should keep ports in local development consistent. I suggest moving to 6878.

Reply:
According to the check out branch, this change is already made in the following lines:
  redis:
    ports: 
      - "6878:6379"